### ISSUE TITLE
Support replication at Volume Groups level

### ISSUE THREAD

**Issue by ELENAGER** [csi-addons/spec#45]
Kubernetes and the CSI Spec does not have Volume Groups support yet. Once that feature is available, csi-addons project probably want to extend the volume replication to support it too. 

This issue is opened for tracking

**Comment by Madhu-1** [comment]
Yes once the feature is available in the csi specification we will add new RPC for volume replication also

**Comment by Nikhil-Ladha** [comment]
I think we are good to close this issue now, @Madhu-1 ?
